### DISCUSSION TITLE
Manipulation of the Document Object Model of the platform in AngularJs;

### DISCUSSION THREAD

**Discussion by prishnah** [clovercommunity/resco#6] [Q&A] [answered]
Often times manipulating the DOM directly from the controller itself.
Since manipulating the DOM is such a common task when using this platform;
actions such as rendering SVG files, refreshing a page's title based on a context change or setting the focus on a control after a validation error and et cetera.

**Comment by albertcodes** [comment, marked as answer]
Restraining yourself from manipulating the DOM directly, using jQuery, the global document object or the **_ElementRef_**.nativeElement.
Instead, use the _**Renderer2**_ service

**Reply by davatas** [reply]
Correct ✔️
or Renderer for Angular 2

**Reply by prishnah** [reply]
Nail bitter.... this is very helpful 🙏

**Comment by davatas** [comment]
Angular is growing rapidly as a web framework
